sector-builder/src/metadata.rs

```diff
@@ -41,6 +43,12 @@ pub enum SealStatus {
     Sealing,
 }
 
+#[derive(Clone, Serialize, Default, Deserialize, Debug, PartialEq)]
+pub struct SealTicket {
+    pub height: u64,
```

Comment:
please add a comment, to what `height` refers to

Reply:
done